pkg/account/server.go

```diff
@@ -42,6 +42,12 @@ type Server interface {
 	Logout(w http.ResponseWriter, r *http.Request)
 }
 
+// Config represents the Account app configuration
+type Config struct {
+	OAuth   oauth.Config `name:"oauth"`
+	Session sess.Config  `name:"session"`
+}
```

Comment:
We typically put config structs in `config.go` so that the right CODEOWNERS are requested for review when making changes that could affect config compatibility.